lib/engine/game/g_1847_ae/round/draft.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require_relative '../../../round/draft'
+
+module Engine
+  module Game
+    module G1847AE
+      module Round
+        class Draft < Engine::Round::Draft
+          def initialize(game, steps, **opts)
+            super
+
+            @players_reordered_for_next_round = false
+          end
+
+          def next_entity_index!
+            # First round of draft is perforemd in reverse player order, then it must be reversed back to normal
+            if @entity_index == @entities.size - 1 && !@game.draft_first_round_finished
```

Comment:
Instead of adding the `draft_first_round_finished` to the Game class, you could just check `@reverse_order` and set that to false at the end of the first draft round.